app/src/main/java/com/halim/adam/cashmaster/EditCategory.java

```diff
@@ -0,0 +1,48 @@
+package com.halim.adam.cashmaster;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.halim.adam.cashmaster.Objects.Category;
+
+public class EditCategory extends Activity {
+    Category category = new Category();
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_edit_category);
+
+        TextView nameText = findViewById(R.id.nameText);
+
+        DatabaseHelper dbHelper = new DatabaseHelper(this);
+
+        // get extra
+        int selectedId = getIntent().getIntExtra("id", 0);
+        category = dbHelper.GetCategory(selectedId);
+
+        nameText.append(category.getName());
+    }
+
+    public void GetInput(View view){
```

Comment:
Yeah, this method named `GetInput` everywhere are confusing me. A better naming scheme is to name this method to what it supposed to do. For example in this part you want to update category, so `UpdateCategory` would work better